va/va.h

```diff
@@ -4182,6 +4182,37 @@ VAStatus vaDeriveImage (
     VAImage *image	/* out */
 );
 
+typedef struct _VAAppSurfaceResInfo{
+    uint32_t image_width;
+    uint32_t image_height;
+    uint32_t total_size;
+    uint32_t num_planes;
+    uint32_t pitch[4];
```

Comment:
why 4 if the rest of VAAPI uses 3?

Reply:
VASurfaceAttribExternalBuffers is using 4, 4 element for some planar format? such as A, Y, U, V planar, I am not sure. but anyway, reserved more is flexible than 3.